production/docker/README.md

```diff
@@ -0,0 +1,37 @@
+# Loki cluster using docker-compose
+
+To deploy a cluster of loki nodes on a local machine (as shown below), you could use the `docker-compose-ha-member.yaml` file.
+
+<img src="./docker-compose-ha-diagram.png" width="850">
+
+Some features of the deployment:
+
+- Backend: 3 Loki servers enabled with distributor, ingester, querier module
+- Together they form a cluster ring based on memberlist mechanism (if using consul/etcd, modules can be separate for further separate read/write workloads)
+- Index data are stored and replicated through botldb-shipper
+- Replication_factor=2: the receiving distributor sends log data to 2 ingesters based on consistent hashing
+- Chunk storage is a shared directory mounted from the same host directory (to simulate S3 or gcs)
```

Comment:
@cyriltovena Do we safely support this kind of operations on a local filesystem?

Reply:
I think so.